src/main/kotlin/app/revanced/patches/twelvewidgets/unlock/annotations/DetectionCompatibility.kt

```diff
@@ -0,0 +1,9 @@
+package app.revanced.patches.twelvewidgets.unlock.annotations
+
+import app.revanced.patcher.annotation.Compatibility
+import app.revanced.patcher.annotation.Package
+
+@Compatibility([Package("com.dci.dev.androidtwelvewidgets", arrayOf("1.4.5", "1.4.2"))])
```

Comment:
Is the constraint on the version really required? The patch doesn't seem like it'll break anytime soon.